src/components/Header-pagina-principal-de-receitas.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Link } from 'react-router-dom'
+
+const Header = () => (
+  <div>
+    <Link to="">
```

Comment:
```suggestion
    <Link to="/perfil">
```
a imagem tem que linkar com o perfil